src/test/objectstore/store_test.cc

```diff
@@ -4485,6 +4485,13 @@ TEST(DummyTest, ValueParameterizedTestsAreNotSupportedOnThisPlatform) {}
 
 #endif
 
+static bool plugin_exists(const string& compressor_dir)
+{
+  string path(compressor_dir);
+  path += "/libceph_snappy.so";
```

Comment:
or you'd suggest we check for the existence of the `compressor_dir` directory instead?